students/tim_lurvey/lesson01/Assignment/test_unit.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/env/bin python
+from unittest import TestCase
+from unittest import mock
+
+from inventory_management.furniture_class import Furniture
+from inventory_management.inventory_class import Inventory
+from inventory_management.electric_appliances_class import ElectricAppliances
+from inventory_management import market_prices
+from inventory_management import main
+
+
+class InventoryTests(TestCase):
+
+    def test_init(self):
+        I = Inventory(product_code='1',
+                      description='object for test',
+                      market_price=99.,
+                      rental_price=9., )
+
+        check = {'product_code': '1',
+                 'description': 'object for test',
+                 'market_price': 99.,
+                 'rental_price': 9., }
+
+        self.assertEqual(first=I.return_as_dictionary(),
+                         second=check, )
+
+
+class FurnitureTests(TestCase):
+
+    def test_init(self):
+        F = Furniture(product_code='a1',
+                      description='object for test',
+                      market_price=99.,
+                      rental_price=9.,
+                      material='wood',
+                      size='large')
+
+        check = {'product_code': 'a1',
+                 'description': 'object for test',
+                 'market_price': 99.,
+                 'rental_price': 9.,
+                 'material': 'wood',
+                 'size': 'large'}
+
+        self.assertEqual(first=F.return_as_dictionary(),
+                         second=check, )
+
+
+class ElectricAppliancesTests(TestCase):
+
+    def test_init(self):
+        E = ElectricAppliances(product_code='a1',
+                               description='object for test',
+                               market_price=99.,
+                               rental_price=9.,
+                               brand='GE',
+                               voltage='115/230',
+                               )
+
+        check = {'product_code': 'a1',
+                 'description': 'object for test',
+                 'market_price': 99.,
+                 'rental_price': 9.,
+                 'brand': 'GE',
+                 'voltage': '115/230',
+                 }
+
+        self.assertEqual(first=E.return_as_dictionary(),
+                         second=check, )
+
+
+class market_pricesTest(TestCase):
+
+    def test_get_latest_price(self):
+        self.assertEqual(24, market_prices.get_latest_price(''))
+
+
+class mainTest(TestCase):
+
+    def test_main_menu_direct(self):
+        self.assertEqual(main.main_menu('1'), main.add_new_item)
+        self.assertEqual(main.main_menu('2'), main.item_info)
+        self.assertEqual(main.main_menu('3'), main.exit_program)
```

Comment:
This will stop your test, since "3" is not a valid option (should be "q"). This currently causes your tests to stop while displaying the main menu.